cypress/elements/catalog/products/products-list.js

```diff
@@ -50,4 +50,7 @@ export const PRODUCTS_LIST = {
   },
   resultsOnPageSelect: "[data-test-id='PaginationRowNumberSelect']",
   rowNumberOption: "[data-test-id='rowNumberOption']",
+  tileViewButton: '[data-test-id="tileViewButton"]',
+  datagridViewButton: '[data-test-id="datagridViewButton"]',
+  tileProductsView: '[data-test-id="tileView"]',
```

Comment:
I noticed we use different conventions in each file: camel case or kebab case - what do you think to stick to kebab case `menu-item-label-discounts`?